neps/nep-0568.md

```diff
@@ -428,27 +474,22 @@ The state sync algorithm defines a `sync_hash` that is used in many parts of the
 
 ## Consequences
 
-```text
-[This section describes the consequences, after applying the decision. All consequences should be summarized here, not just the "positive" ones. Record any concerns raised throughout the NEP discussion.]
-```
-
 ### Positive
 
-* p1
+* The protocol is able to execute resharding even while only a fraction of nodes track the split shard.
+* New resharding can happen in the matter of minutes instead of hours.
 
 ### Neutral
 
-* n1
+N/A
 
 ### Negative
 
-* n1
+* The storage components need to handle additional complexity of controlling the shard layout change.
 
 ### Backwards Compatibility
 
-```text
-[All NEPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. Author must explain a proposes to deal with these incompatibilities. Submissions without a sufficient backwards compatibility treatise may be rejected outright.]
-```
+Approach is fully backwards compatible, just adding new protocol upgrade on top of existing implementation. Also, we were able to completely remove previous resharding logic, as it was already approved by validators, and to process chunks from any layout, it is enough to take state from that layout from archival node.
```

Comment:
We retained the capability to replay reshardingV2 in archive nodes, not sure if it's worth mentioning

Reply:
I just deleted mention of this, as I don't have enough understanding.